cmd/query/main.go

```diff
@@ -43,6 +44,8 @@ import (
 func main() {
 	logger, _ := zap.NewProduction()
 	casOptions := casFlags.NewOptions("cassandra", "cassandra.archive")
+	// currently no archive for es
+	esOptions := esFlags.NewOptions("es")
```

Comment:
although even for Cassandra I don't think we're actually wiring in the archive storage